Makefile

```diff
@@ -16,3 +20,7 @@ FILES = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 format:                         ## Format source code.
 	gofmt -w -s $(FILES)
 	goimports -local github.com/Percona-Lab/pmm-api-tests -l -w $(FILES)
+
+clean:
+	rm -f ./pmm-api-tests-output.txt
+	rm -f ./pmm-api-tests-junit-report.xml
```

Comment:
Those files are already ignored, why have a separate target for them?

Reply:
Just good practice to have a target for clean stuff. But yes it doesn't make any sense in this context. At least for Jenkins.

Reply:
`make clean` become a good practice at the time before out-of-tree builds become mainstream and VCS become mandatory. We have both, so we can use `git clean [-x]`.